src/api/NotificationsAssistant.ts

```diff
@@ -142,7 +159,7 @@ export default class NotificationsAssistant{
 			var time = startMillisecond
 
 			for(let f=0; f < this.frequency; ++f){
-
+				console.log(f)
 				// retrieve random verse content
 				var randomVerse = this.verses[this.getRandomIndex(this.verses.length)]
 				var verseTitle = "Proverbs " + randomVerse.Chapter + ":" + randomVerse.VerseNumber
```

Comment:
**This is where you need to add your notification schedule guards: the thing that prevents notifications from being scheduled in the past.**